CONTRIBUTING.md

```diff
@@ -113,6 +113,11 @@ following rules before you submit a pull request:
    scale in dimensionality: n_features is expected to be lower than
    100".
 
+-  If your PR includes changes to files under the ``doc/`` or
```

Comment:
That's a good point, I use it on both and haven't run into an issue specific to either. 

Great, fixed up and submitted. Thanks for your help!